Cargo.toml

```diff
@@ -30,8 +30,8 @@ borsh = { version = "0.10.3", default-features = false }
 bytemuck = { version = "1.14", default-features = false }
 derive_more = "0.99.17"
 hex-literal = "0.4.1"
-ibc = { version = "0.47.0", default-features = false, features = ["serde", "borsh"] }
-ibc-proto = { version = "0.37.1", default-features = false, features = ["serde"] }
+ibc = { version = "0.47.0", default-features = false, features = ["borsh"] }
```

Comment:
I think the `ibc::applications::transfer` requires the `serde` feature. So we have to keep the `serde` feature

Reply:
Yes, you’re right. It’s weird that `cargo test` run didn’t fail compiling the code.

Reply:
OK, actually, it’s probably because some dev dependency enables serde feature.

Reply:
oh damn, got it